### ISSUE TITLE
[EN][Dungeon-WotLK] Drak'Tharon Keep

### ISSUE THREAD

**Issue by Tj20** [Atlantiss/BugTracker#2748]
http://www.wowhead.com/zone=4196/draktharon-keep
The instance works remarkably well, only the first boss is easier than expected.

### [Trollgore](http://www.wowhead.com/npc=26630/trollgore#encounter)
- does have [Drakkari Invader](http://www.wowhead.com/npc=27754) adds, but they are unkillable and just stand/hang around, doing nothing ([combat log](http://pastebin.com/hdXLvYhM), [video](https://youtu.be/LSPVJP1WrUg?t=3m28s))
- does not cast [Infected Wound](http://www.wowhead.com/spell=49637) 
- does cast a [Corpse Explode](http://www.wowhead.com/spell=49555) buff on the Invaders, but it does nothing
- does cast [Consume](http://www.wowhead.com/spell=49381) which deals damage to the group as expected, but doesn't give Trollgore a stackable damage buff (stays at one stack)

![pic](http://i.imgur.com/InAYViw.jpg)

### Trash Mobs
- There's a room with mobs named "Cosmetic Drakkari Bat [PH]" ([pic](http://i.imgur.com/aBNetYE.jpg)). According to this [video](https://youtu.be/LSPVJP1WrUg?t=8m10s) it should contain some [Drakkari Bats](http://www.wowhead.com/npc=26622) and one [Risen Drakkari Bat Rider](http://www.wowhead.com/npc=26638/risen-drakkari-bat-rider) I think.
- The terrace before the endboss has 1 [Risen Drakkari Death Knight](http://www.wowhead.com/npc=26830) ([pic](http://i.imgur.com/7AIhRdF.jpg)) where there would be 2 groups á 2 and a patrolling one normally ([video](https://youtu.be/LSPVJP1WrUg?t=11m20s))

**Comment by Varchord** [comment]
Novos the Summoner is working properly now? I remembered his event being bugged (he had perma immunity)

**Comment by Tj20** [comment]
Yes, he worked just like I remember him, in fact, all the other bosses did. Novos' stairs event (his shield dropped properly after killing the 4 casters), King Dred's fear and tank debuffs, Tharon'ja transforming you in a skeleton with special abilities... I was impressed.

**Comment by Varchord** [comment]
And what about the last boss?
I remember that if you kill him in his "alive" form he doesn't droop loot

**Comment by Tj20** [comment]
I don't know... we killed him in the normal phase when we are alive and he is a skeleton, and so I didn't notice anything unusual there.
